### PR TITLE
HAWQ-873. improve checking time for .travis.yml

### PR DESCRIPTION
**Pull request by xunzhang**
From 25min -> 17min: https://travis-ci.org/xunzhang/incubator-hawq/builds/140718129

### CODE REVIEW

**Comment by xunzhang** [comment]
cc @liming01 @yaoj2 @radarwave

**Comment by yaoj2** [comment]
Cool +1

**Comment by radarwave** [comment]
LGTM. +1

File: .travis.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ before_script:
   - ./configure
 
 script:
-  - make -j2
+  - make -j$(sysctl -n hw.ncpu)
```
Comment (ictmalili) [inline .travis.yml]: @xunzhang ,  could you tell what's the meaning for this parameter "hw.ncpu"? Have you uploaded the code to HAWQ master code? Thanks

Comment (xunzhang) [inline .travis.yml]: It is the number of CPUs in the platform like `nprocs` in ubuntu. This pr has already been merged into master.

**Pull request merged**